content/actions/security-for-github-actions/security-hardening-your-deployments/configuring-openid-connect-in-amazon-web-services.md

```diff
@@ -124,9 +124,9 @@ To update your workflows for OIDC, you will need to make two changes to your YAM
 
 The `aws-actions/configure-aws-credentials` action receives a JWT from the {% data variables.product.prodname_dotcom %} OIDC provider, and then requests an access token from AWS. For more information, see the AWS [documentation](https://github.com/aws-actions/configure-aws-credentials).
 
-* `<example-bucket-name>`: Add the name of your S3 bucket here.
-* `<role-to-assume>`: Replace the example with your AWS role.
-* `<example-aws-region>`: Add the name of your AWS region here.
+* `BUCKET-NAME`: Add the name of your S3 bucket here.
+* `AWS-REGION`: Add the name of your AWS region here.
+* `ROLE-TO-ASSUME`: Replace this with your AWS role. For example, `arn:aws:iam::1234567890:role/example-role`
```

Comment:
Made `<role-to-assume>` a placeholder `ROLE-TO-ASSUME` with an example, and referenced that in the YAML file similar to what's in GCP documentation:

![image](https://github.com/user-attachments/assets/6a777a24-5cb2-4a0e-a0e8-ae1b7391f9f1)